scss/_nav.scss

```diff
@@ -91,6 +91,8 @@
 //
 
 .nav-pills {
+  @include clearfix();
+  
```

Comment:
Please remove the trailing whitespace from this line; see the Travis build failure.